### DISCUSSION TITLE
Triggerbot independent from Aimbot

### DISCUSSION THREAD

**Discussion by diekunstderfuge** [danielkrupinski/Osiris#3292] [Ideas]
Would it be possible for the triggerbot to have completely independent adjustments (i.e. activating key, fov, aimlock, bone, etc) from the regular aimbot?

Suppose one config that I activate the aimbot on key press by using the left mouse button. And at the same time I activate the triggerbot by holding the third mouse button.

In this case, since the aimbot is only activated on press of the left mouse button, if I were to hold the third mouse button I would only activate the triggerbot, but I would be shooting without any aim assistance. In this case, the triggerbot would be almost useless.

Now, if the tiggerbot were independent from the aimbot, it would have it's own activation, aimlock, fov and bone adjustments (and others adjustments)

In this case I could decide if I was going to shoot with the aimbot or with the triggerbot+aimlock just by using different keys (in the example, aimbot with the mouse left button or triggerbot+aimlock by holding the third mouse button).

**Comment by diekunstderfuge** [comment]
Forget that!
I can achieve the same effect if the aimbot could be activated by two different keys. For instance, mouse button 1 or mouse button 3.

Or, if the triggerbot activates the aimbot by itself when I'm using it. 

How can I do that?

**Reply by aoqia194** [reply]
You could create a checkbox in the Aimbot and also Triggerbot menu such as "Link with Triggerbot" (For Aimbot tab) and "Link with Aimbot" (For Triggerbot tab)".
In the code, when the code is ran, if "Link with Triggerbot" is checked from the Aimbot tab, first set the Triggerbot checkbox to true as you don't want to be able to set the hotkey twice (no reason to), and then also set the Triggerbot hotkey.
If this makes any sense.
